scheduler/src/cook/config.clj

```diff
@@ -355,7 +354,6 @@
   (try
     (let [literal-config {:config config-map}]
       (pre-configuration literal-config)
-      (.println System/err "Configured logging")
```

Comment:
Shouldn't be part of this PR :)

Reply:
It's related to this PR because I found it very annoying to run the unit tests with these `println`s, and I don't think they're necessary anyway